backend/app/controllers/api/v1/cuisines_controller.rb

```
@@ -0,0 +1,67 @@
+module Api
+  module V1
+    class CuisinesController < ApplicationController
+      before_action :set_cuisine, only: [:show, :update, :destroy]
+
+      # GET /cuisines
+      def index
+        render json: Cuisine.all, status: :ok
+      end
+
+      # GET /cuisines/1
+      def show
+        if @cuisine
+          render json: @cuisine, status: :ok
+        else
+          render status: :unprocessable_entity
+        end
+      end
+
+      # POST /cuisines
+      def create
+        @cuisine = Cuisine.new(cuisine_params)
+        if @cuisine.save
+          render json: @cuisine, status: :created
+        else
+          render json: {errors: @cuisine.errors}, status: :unprocessable_entity
+        end
+      end
+
+      # PATCH/PUT /cuisines/1
+      def update
+        if @cuisine
+          if @cuisine.update(cuisine_params)
```

How about a "fail fast" approach?
Like failing first with
`render json: {message: 'cuisine does not exist'}, status: :unprocessable_entity unless @cuisine` 
and then go on to the rest of the code. (same in case update fails)